src/components/20-molecules/datepicker/pw.ui.test.js

```diff
@@ -0,0 +1,57 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+
+describe('Datepicker', () => {
```

Comment:
I really like your effort to migrate over to playwright! Nice!

Reply:
Have to:)

Reply:
I do like your efforts, too - but I don't like that you are doing it in a PR that is totally unrelated.

I wrote a document on traceability of PL changes, and this would adversely affect it. You are fixing 2 button issues here - other issues should be separated out into a properly named separate PR.

Reply:
Just look closer to the tests i have migrated. The tests are clicking on a axa-button to do something. And my change is that the button only accepts click events which are `isTrusted`. And testcafe generates synthetic events which are isTrusted=false. Thats why i have to fix/migrate failing tests.